Mathlib/Analysis/SpecialFunctions/PolarCoord.lean

```diff
@@ -185,4 +200,10 @@ protected theorem integral_comp_polarCoord_symm {E : Type*} [NormedAddCommGroup
     measurableEquivRealProd.symm.measurableEmbedding, ← integral_comp_polarCoord_symm]
   rfl
 
+protected theorem lintegral_comp_polarCoord_symm (f : ℂ → ENNReal) (hf : Measurable f) :
```

Comment:
```suggestion
protected theorem lintegral_comp_polarCoord_symm (f : ℂ → ℝ≥0∞) (hf : Measurable f) :
```
Also, I am surprised that you would need `f` to be measurable, while it's not needed in the previous one. Use the fact that you have a measure isomorphism to get rid of it?

Reply:
Indeed, `Measurable f` is not necessary. Thanks a lot for the review!